securedrop_client/queue.py

```diff
@@ -61,11 +62,6 @@ class RunnableQueue(QObject):
     '''
     resume = pyqtSignal()
 
-    """
-    Signal emitted when the queue successfully.
-    """
-    pinged = pyqtSignal()
```

Comment:
removed the `pinged` signal because we already have a `resume` signal that is emitted when the user clicks "Retry" to restart the queues or when the continuous background sync succeeds and calls resume_queues in it's success handler

Reply:
ah it looks like we're still attempting to emit that signal in `RunnableQueue.process`